client/src/components/Project/Project.js

```diff
@@ -46,11 +46,19 @@ class Project extends Component {
     )
   }
 
+  getUserTypeContact () {
```

Comment:
A better name for this would be that indicates it returns a boolean, something like `isUserTypeContact`. Also, this can be re-used on lines 27 and 33!